pages/linux/phar.md

```diff
@@ -0,0 +1,35 @@
+# phar
+
+> Create, update or extract PHP archives (PHAR).
+
+- Add space-separated files or directories to a Phar file:
+
+`phar add -f {{path/to/phar}} {{files_or_directories}}`
+
+- Delete the specified file or directory from a Phar file:
+
+`phar delete -f {{path/to/phar}} -e {{file_or_directory}}`
+
+- Display full usage information and available hashing/compression algorithms:
+
+`phar help`
+
+- Compress or uncompress files and directories in a Phar file:
+
+`phar compress -f {{path/to/phar}} -c {{algorithm}}`
```

Comment:
How do you tell whether it's compressing or decompressing?

Reply:
Maybe that's a bit confusing.

Basically, to compress you'd use:
`phar compress -f {{path/to/phar}} -c {{auto|gz|gzip|bz2|bzip2}}`

To decompress you'd use:
`phar compress -f {{path/to/phar}} -c {{none|0}}`

Reply:
Ahh, right. Hrm, not sure how to work that into the description

Reply:
Neither am I. :thinking: